\chapter{Obesity associated genetic signatures and cancer}
\label{cha:obesity_genetic_signatures_and_cancer}

The obesity associated genetic signatures are central to this project in order to clarify the relationship between \gls{bmi} and cancer.
Firstly in this chapter, the previously identified obesity associated signatures from the studies conducted by \citet{Creighton2012} and \citet{Fuentes-Mattei2014} were examined in turn to judge the agreement of these signatures with the sample \gls{bmi}, as presented in thier results.
Secondly, novel obesity associated genetic signatures were identified in the \citet{Creighton2012} data set and was compared with the signatures.
Lastly, the presence of common genes or pathways that were associated with obesity in multiple types of cancer was explored using the \gls{icgc} cancer data sets.

\section{Obesity associated genetic signature from \citet{Creighton2012} study}
\label{sec:creighton_obesity_metagene}

Obesity metagene was created using the obesity associated genetic signatures from \citet{Creighton2012} study.
% As the genetic signatures were described as gene probe IDs, the gene probes had to be converted into gene symbols (see \cref{sec:data}).
% This resulted in 644 unique gene symbols, compared to 799 gene probes IDs in the original signature.
There was no description about the normalisation method used by Creighton's group when they first analysed their data, so the more reliable \gls{rma} method was used to  normalise the Creighton data set.
The obesity metagene was plotted in a heatmap with the sample gene expression (\cref{fig:crmetaheat1}) to check whether the metagene scores were in accordance with the overall gene expression of the samples.

\begin{figure}[htb]
	\centering
	\includegraphics[width=0.8\linewidth]{results1/crmeta_original}
	\caption[Heatmap of Creighton's metagene in Creighton's data]{Heatmap showing Creighton's obesity associated metagene with the gene expressions of obesity associated genes in Creighton's data. Level of expression is represented in the top right histogram, where low and high gene expression were colour-coded with blue and red, respectively. Each row of the heatmap represents a gene from the obesity associated genetic signature, and each column of the heatmap represents a sample from Creighton's data. The obesity associated metagene scores of the samples are shown in a separate row at top of the heatmap, and the tree diagram of the heirarchical clustering of the genes is shown to the left of the heatmap. For clarity, the metagene scores were flipped in order to match the results from \citet{Creighton2012} paper.}
	\label{fig:crmetaheat1}
\end{figure}

As shown in \cref{fig:crmetaheat1}, high obesity associated metagene score of the sample reflected low expression in majority of the genes in the signature, and in contrast, low obesity associated metagene score of the sample reflected high expression in majority of the genes in the signature.
This was consistent with the reported property of the obesity associated signature by \citet{Creighton2012}.
To provide further evidence that the obesity associated metagenes were in fact associated with the \gls{bmi} status and \gls{bmi} value of the samples, a box plot and a scatter plot were created, respectively (\cref{fig:crmetaboxplot1}).

\begin{figure}[htp!]
	\centering
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\hfill
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\caption[Creighton metagene and sample \gls{bmi}/\gls{bmi} status]{(insert figure of box plot and scatter plot)}
	\label{fig:crmetaboxplot1}
\end{figure}

\cref{fig:crmetaboxplot1} clearly showed that the obesity associated metagene from \citet{Creighton2012} study significantly associated with the sample \gls{bmi} status, as well as the \gls{bmi} value of the samples (include p-value and r-squared).
It should be noted here that the obesity associated metagene significantly associated with the samples that were obese, but not the samples that were overweight.
This was due to the fact that the obesity associated genetic signatures were originally identified from the comparison of the samples that were obese with the samples that were not obese, and therefore the metagene scores were significant with obese group, but not the overweight group.
\\

\noindent
Now that the association of the obesity metagene from \citet{Creighton2012} study was established in Creighton's data set, the obesity associated signature was transferred to the \gls{icgc} cancer data.
The direction of the obesity associated metagene was checked in the Creighton's data, so that high metagene scores reflected high sample \gls{bmi} and low metagene scores reflected low sample \gls{bmi} (\cref{fig:crmetaboxplot1}).
The transformation matrix was created in Creighton's data, as described in \cref{sub:svd}.

All of the \gls{icgc} data were normalised as described in \cref{ssub:rna_seq_data}.
Before the transformation matrix was applied to the log$_{10}$-normalised cancer data, the suitability of standardised data or untouched (non-standardised) data was determined (appendix).
From these results, the standardised data was the most suitable to use for matrix transformation.
The transformation matrix was applied to each cancer data set in turn to obtain the obesity metagenes for each of the data set.
Each obesity metagenes were plotted on a heatmap with the corresponding data set in which the metagene was taken from \cref{fig:crmetaicgc1}.
These heatmaps confirmed that the obesity associated metagene derived from Creighton's data was able to capture the overall gene expression pattern, where the metagene scores reflected the expression levels of majority of the genes in the signature.

\begin{figure}[htp!]
	\centering
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\hfill
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\caption[Creighton metagene and \acrshort{icgc} cancer data]{(insert ICGC heatmaps)}
	\label{fig:crmetaicgc1}
\end{figure}

\begin{figure}[htp!]
	\centering
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\hfill
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\caption[Creighton metagene and sample \gls{bmi}/\gls{bmi} status in \acrshort{icgc} data]{(insert figure of box plot and scatter plot for ICGC data)}
	\label{fig:crmetaboxicgc1}
\end{figure}

As before, association of the obesity metagene with the sample \gls{bmi} and \gls{bmi} status was checked in their respective cancer data set (\cref{fig:crmetaboxicgc1}).
In contrast to the strong association of the obesity metagene with sample \gls{bmi} and \gls{bmi} status in Creighton's data, the obesity metagene taken from the \gls{icgc} cancer data did not significantly associate with the sample \gls{bmi} or \gls{bmi} status in any of the \gls{icgc} cancer data.

There could be several reasons for the apparent lack of association of the metagene with the sample \gls{bmi} or \gls{bmi} status.
First, transformation matrix was derived from Creighton's microarray data, but the \gls{icgc} cancer data were \gls{rnaseq} data.
Though the log$_{10}$ normalisation and standardisation of the data was the most appropriate adjustment to be made to the \gls{rnaseq} data, this adjustment may not be equivalent to the \gls{rma} normalisation method that was used on the microarray data.
Secondly, none of the \gls{icgc} cancer was from breast as in Creighton's data.
Since the obesity associated signature was identified in breast cancer data, the signature may be specific to breast cancer and may not be suitable in other cancer types.
\\

\noindent
To check whether the metagene was specific to breast cancer microarray data, the same transformation matrix was applied to Print's breast cancer microarray data.
Print's data was normalised with \gls{rma} method as described in \cref{ssub:microarray_data}, and the transformation matrix was applied to the normalised data to obtain the metagene.
The metagene was again compared with the gene expression of the samples with a heatmap (\cref{fig:crmetaprint1}), and the association of the metagene with the sample \gls{bmi} and \gls{bmi} status was examined with box and scatter plots (\cref{fig:crmetaboxprint1}).

\begin{figure}[htp!]
	\centering
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\hfill
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\caption[Creighton metagene and Print's cancer data]{(insert Print heatmaps)}
	\label{fig:crmetaprint1}
\end{figure}

\begin{figure}[htp!]
	\centering
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\hfill
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\caption[Creighton metagene and sample \gls{bmi}/\gls{bmi} status in Print's data]{(insert figure of box plot and scatter plot for Print data)}
	\label{fig:crmetaboxprint1}
\end{figure}

The obesity metagene managed to reflect the overall gene expression of the samples in Print's data.
However, like in \gls{icgc} cancer data, the obesity metagene scores did not significantly associate with the sample \gls{bmi} or \gls{bmi} status.
These results confirmed that the lack of association of the obesity metagene from Creighton's data was not due to the technology in which the data was gathered (mcroarray or \gls{rnaseq}), nor the type of cancer in which the transformation matrix was applied to.

All together, all of these results suggest that the obesity metagene identified in the study conducted by \citet{Creighton2012} associated with sample \gls{bmi} and \gls{bmi} status in Creighton's data, but did not transfer well in other cancer data sets.
The lack of association with \gls{bmi} was not due to the type of technology platform in which the data was gathered, as neither the \gls{icgc} \gls{rnaseq} data nor Print's microarray data did not show significant association with the obesity metagene.
Furthermore, Creighton's metagene was not dependent on the cancer type since the obesity associated genetic signature did not show significant association in neither the \gls{icgc} cancer data nor in Print's breast cancer data.

One possible reason why Creighton's obesity metagene did not show significant association with other data sets could be because the genetic signature was in fact not an obesity specific signature, but a signature that detected another clinical variable.
Another reason for this apparent lack of association could be that the genetic signature was too specific to the data and was not a broad obesity associated genetic signature, but an obesity associated signature specifically for Creighton's data set.

\section{Obesity associated genetic signature from \citet{Fuentes-Mattei2014} study}
\label{sec:fm_obesity_metagene}

Since Creighton's obesity associated genetic signature was not associated with sample \gls{bmi} or \gls{bmi} status in other data sets, Fuentes-Mattei's (FM) obesity associated genetic signature was examined to see whether FM's obesity metagene associated with sample \gls{bmi} or \gls{bmi} status.
Since FM's data set did not have sample \gls{bmi} information, FM's obesity metagene was not able to be compared with the sample \gls{bmi} or \gls{bmi} status in the original FM's data.
However, the transformation matrix was still applied to other cancer data to see whether FM's obesity metagene associated with sample \gls{bmi}/\gls{bmi} status

Firstly, FM's data was normalised with the \gls{rma} method and \gls{svd} was applied to the normalised data to get the transformation matrix.
The transformation matrix was used to transform the \gls{rma}-normalised Creighton's data to extract FM's obesity metagene scores.
FM's obesity metagene scores were compared with gene expressions and sample \gls{bmi}/\gls{bmi} status in Creighton's data, as shown in \cref{fig:fmmetacr1,fig:fmmetaboxcr1}.
Clearly, like with Creighton's obesity metagene, FM's obesity metagene was reflective of the overall gene expression of the samples, but did not associate with the sample \gls{bmi} or \gls{bmi} status.

\begin{figure}[htp!]
	\centering
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\hfill
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\caption[FM metagene and Creighton's cancer data]{(insert Creighton heatmaps)}
	\label{fig:fmmetacr1}
\end{figure}

\begin{figure}[htp!]
	\centering
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\hfill
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\caption[FM metagene and sample \gls{bmi}/\gls{bmi} status in Creighton's data]{(insert figure of box plot and scatter plot for Creighton data)}
	\label{fig:fmmetaboxcr1}
\end{figure}

Next, the transformation matrix was applied to the \gls{icgc} cancer data and resulting metagenes were compared with the gene expression and sample \gls{bmi}/\gls{bmi} status.
As evident in \cref{fig:fmmetaicgc1}, FM's obesity metagene scores appeared to reflect the gene expression of FM's obesity associated genetic signature.
However, as with all the results in this section so far, FM's obesity metagene did not significantly associate with any of the \gls{icgc} cancer data (\cref{fig:fmmetaboxicgc1}).

\begin{figure}[htp!]
	\centering
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\hfill
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\caption[FM metagene and \gls{icgc} cancer data]{(insert ICGC heatmaps)}
	\label{fig:fmmetaicgc1}
\end{figure}

\begin{figure}[htp!]
	\centering
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\hfill
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\caption[FM metagene and sample \gls{bmi}/\gls{bmi} status in \gls{icgc} data]{(insert figure of box plot and scatter plot for Creighton data)}
	\label{fig:fmmetaboxicgc1}
\end{figure}

Lastly, the transformation matrix was applied to Print's data (\cref{fig:fmmetaboxcris1,fig:fmmetacris1}).
Again, FM's obesity metagene scores reflected the gene expression of the samples, did not significantly associate with sample \gls{bmi}/\gls{bmi} status.
\\

\begin{figure}[htp!]
	\centering
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\hfill
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\caption[FM metagene and Print's cancer data]{(insert Print heatmaps)}
	\label{fig:fmmetacris1}
\end{figure}

\begin{figure}[htp!]
	\centering
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\hfill
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\caption[FM metagene and sample \gls{bmi}/\gls{bmi} status in Print's data]{(insert figure of box plot and scatter plot for Print's data)}
	\label{fig:fmmetaboxcris1}
\end{figure}

\noindent
These results showed that FM's obesity associated metagene was not transferable to other cancer data sets, similar to Creighton's obesity associated metagene.
This meant that neither Creighton's nor FM's obesity associate metagenes may be too specific to the orignial data set in which the signature was identified in.
Furthermore, it may be possible that these obesity associated metagenes may not be related to obesity, but another clinical variable which may be closely related to \gls{bmi}.

\section{Novel obesity associated genetic signatures from \citet{Creighton2012} data set}
\label{sec:creighton_obesity_metagene_new}

\subsection{Identification of obesity associated genetic signatures}
\label{sub:identification_of_obesity_associated_genetic_signatures}

Both metagenes generated from Creighton's data and FM's data were able to capture the overall pattern of gene expression of the samples, but did not associate with sample \gls{bmi}/\gls{bmi} status.
One possible reason for this result could be that the obesity associated genetic signatures from Creighton's and FM's study may not have been truly associated with sample \gls{bmi} status, but another clinical variable.
To remove this possibility, obesity associated genetic signatures were identified in Creighton's data after controlling for all the clinical variables in the data set.
FM's data set was not used to get obesity associated genetic signatures, as no \gls{bmi} information was available for the samples in FM's data.

Firstly, to validate whether the identified obesity associated genetic signature was similar to the signature that Creighton's groups had originally found, the identified \glspl{deg} were compared with the set of genes identified by Creighton's group.
\glspl{deg}  were identified between the samples from obese patient and non-obese patient in the \gls{rma}-normalised Creighton's data, as described in \cref{sec:gene_expression_analysis}.
Without adjusting the p-value with FDR, 5278 gene probes and 1781 gene probes were significant at p \textless{} 0.05 and p \textless{} 0.01, respectively.
After afjustment, there were only 9 gene probes significant at p \textless{} 0.05 and no gene was significant at p \textless{} 0.01.
Furthermore, there were only 61 gene probes that were significantly differentially expressed at p \textless{} 0.05 with a log$_2$ \gls{fc} greater than 1.2.
From these observations, the log$_2$ fold change of the gene probes were ignored and the threshold p-value was set to 0.01 (without adjustment) for the identification of gene probes.
This was to include as many gene probes as possible without losing power (?).
Additionally, when there were more than 799 gene probes identified, only the most significant 799 gene probes were taken as the obesity associated genetic signature, as this many genes were originally identified by Creighton's group.
These criteria were applied for the identification of other genetic signatures as well.

The above analysis was repeated with the residual data (\gls{rma}-normalised Creighton's data that had been controlled for other clinical variables; see \cref{sub:residual_data_creation}).
The clinical variables controlled were age, ethnicity, menopause status, tumour grade, hormone (\gls{er}, \gls{pr}, and \gls{her2}) statuses  and \gls{ln} status.
In the residual data, 1104 gene probes were significant with unadjusted p-value (p \textless{} 0.01).
Again, the most significant 799 genes were taken as the obesity associated genetic signature from this data.

In addition to the above two obesity associated genetic signatures, two more sets of genetic signatures were identified by taking the common genes between the above two signatures with Creighton's original obesity associated genetic signatures.
There were 239 common gene probes between Creighton's original genetic signature and the gene probes identified from the unadjusted Creighton's data, and 168 common gene probes between the original signature and gene probes from residual data in Creighton's data.
The genetic signatures identified were summarised as a Venn diagram, as shown in \cref{fig:venn1}.
\\

\begin{figure}[htp!]
	\centering
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\caption[Summary of the \glspl{deg} identified from Creighton's data]{(add venn diagram)}
	\label{fig:venn1}
\end{figure}

\noindent
There was a possible bias between different ethnic groups, where African American patients were more likely to be obese compared to the Caucasian patients (see \cref{ssub:creighton_study}).
Though ethnicity was controlled in the residual data, the effect of ethnicity on the data was completely removed to prevent any possibility of ethnicity influencing the analysis.
Therefore, the effect of ethnicity was completely removed by considering only the Caucasian patients in Creighton's data, which left a total of 77 Caucasian patients in the data set.
With this Caucasian-only data set, obesity associated genetic signatures were identified as described above; first with unadjusted data, then with the data with clinical variables adjusted (except ethnicity, as it has already been controlled for), and lastly the common genes with Creighton's original signature were identified.

2129 gene probes and 1558 gene probes were identified in the unadjusted and clinical variable-adjusted Caucasian patient data, respectively.
As before, the most significant 799 gene probes were selected from these gene probes.
There were 148 and 92 common gene probes with Creighton's original genetic signatures and unadjusted or clinical variable-adjusted Caucasian patient data set, respectively.
Again, Venn diagram was used to summarise the genes identified from the Caucasian patient data set (\cref{fig:venn2}).
\\

\begin{figure}[htp!]
	\centering
	\includegraphics[width=0.45\linewidth]{results1/crmeta_original}
	\caption[Summary of the \glspl{deg} identified from Creighton's data (Caucasian patient data)]{(add venn diagram)}
	\label{fig:venn2}
\end{figure}

\noindent
For simplicity, Creighton's original obesity associated signature will be referred to as ``original'' signature, new signatures from unadjusted and clinical variable-adjusted Creighton's data as ``Cr'' and ``Res'' signature respectively, and the signatures from unadjusted and clinical variable-adjusted Caucasian patient data set as ``Ca'' and ``CaRes'' signature respectively.
All of the overlap signatures will be referred to as their respective signature, but with ``Ol'' added at the end, for example ``CaResOl'' for the overlap signature identified in clinical variable-adjusted Caucasian patient data set.
All of these obesity associated genetic signatures (eight in total) were checked to see whether these signatures had significant association with sample \gls{bmi}/\gls{bmi} status.

\subsection{Novel obesity associated signatures and sample \gls{bmi}/\gls{bmi} status}
\label{sub:_novel_obesity_associated_signatures_and_sample_bmi}

As with the original signature, all eight signatures were validated in Creighton's data set first, and then compared in other cancer data sets by using the transformation matrix generated in Creighton's data.
% Before the metagenes were created, all of the signatures were converted into gene symbols, as described in \cref{sec:data} (\cref{tab:signature_gene_num}).
Creighton's data was normalised with \gls{rma} method and \gls{svd} was applied to generate the metagene for the corresponding genetic signature.







% \begin{table}[htbp]
% 	\centering
% 	\caption{Summary of the number of gene probes and genes in each obesity associated genetic signature}
% 	\label{tab:signature_gene_num}
% 	\begin{tabular}{lcc}
% 		Obesity associated genetic signature & No. gene probes   & No. genes\\
% 		\hline
% 		\rule{0pt}{2.25ex}Cr & 799 & 678\\
% 		Res                  & 799 & 655\\
% 		Ca                   & 799 & 657\\
% 		CaRes                & 799 & 651\\
% 		CrOl                 & 239 & 199\\
% 		ResOl                & 168 & 147\\
% 		CaOl                 & 148 & 128\\
% 		CaResOl              &  92 & 86\\
% 		\hline
% 		\hline
% 	\end{tabular}
% \end{table}














\section{Common genes across multiple cancer types}
\label{sec:common_genes_across_multiple_cancer_types}

It was clear that the metagenes created from obesity signatures were not consistent across different data sets.
To identify genes that were associated with sample \gls{bmi} in multiple cancer types, \glspl{deg} were searched (?).







\section{Pathways enriched in obesity associated genes}
\label{sec:pathways_enriched_in_obesity_associated_genes}












