\chapter{Methods (draft)}
\label{ch:methods}

\section{\gls{r} -- statistical programming language}
\label{sec:r}

All statistical analyses and data manipulation were carried out with \gls{r} (version 3.3.2 -- ``Sincere Pumpkin Patch''), a free open-source programming language and software environment for statistical computing and graphics \citep{R2016}.

\section{\Gls{bmi}}
\label{sec:bmi}

\subsection{\gls{bmi} Calculation}
\label{subsec:bmicalc}

Where missing, the \gls{bmi} of the samples were calculated from the clinical data using \cref{eq:bmicalc}.

\begin{equation}
	\label{eq:bmicalc}
	\gls{bmi} = \frac{Weight (kg)}{Height^2(m^2)}\\
\end{equation}

\subsection{\gls{bmi} Classification}
\label{subsec:bmiclassification}

Samples were classified based on the \gls{who} definition, as shown in \cref{tab:whobmiclass}.
\begin{table}[htb]
	\caption{\gls{who} defined \gls{bmi} classification}
	\label{tab:whobmiclass}
	\begin{center}
		\begin{tabular}{lc}
			Classification & \gls{bmi} Value\\
			\hline
			\rule{0pt}{2.25ex}Underweight & \textless{} 20.0\\
			Normal weight/lean & 20.0$\sim$24.9\\
			Overweight & 25.0$\sim$29.9\\
			Obese & $\geq{}$ 30.0\\
			\hline
			\hline
		\end{tabular}
	\end{center}
\end{table}

\section{Publicly available cancer data}
\label{sec:data}

The raw microarray data from \citet{Creighton2012}, \citet{Fuentes-Mattei2014} and \citet{Gatza2010a}  studies were downloaded from the \gls{geo} website.
\gls{rnaseq} and clinical data of multiple different cancer types were downloaded from free and publicly available sources such as \gls{icgc} and \gls{tcga}, respectively.
\\

\noindent
The raw Affymetrix HGU\-133A microarray gene expression data files from the \citet{Creighton2012} study were downloaded from the \gls{geo} database (\gls{geo} accession ID: GSE24185).
Clinical data of the samples (age, ethnicity, tumour grade, menopause status, \gls{bmi}, \gls{er} status, \gls{pr} status, \gls{her2} status, and \gls{ln} status) were obtained from the supplementary table 1 from \citet{Creighton2012} paper.
799 obesity associated gene probes identified in the \citet{Creighton2012} study were obtained from the supplementary data file 1 from \citet{Creighton2012} paper.

The raw Affymetrix HGU\-133A microarray gene expression data files from the  \citet{Fuentes-Mattei2014} study were downloaded from the \gls{geo} database (\gls{geo} accession ID: GSE\-20194).
Clinical data for the samples (age, ethnicity, tumour grade, \gls{er}/\gls{pr}/\gls{her2} statuses, and treatments used) in this study was also downloaded from the \gls{geo} database (same \gls{geo} accession ID).
130 obesity associated gene probes identified by \citet{Fuentes-Mattei2014} were taken from the supplementary table 3 from their paper.

The raw microarray gene expression data files from the \citet{Gatza2010a} study were downloaded from the \gls{geo} database (\gls{geo} accession ID: GSE1456, GSE\-1561, GSE2034, GSE3494, GSE4922, and GSE6596).
Only the Affymetrix HGU\-133A microarray samples were included in this project, as other microarray data were analysed using the Affymetrix HGU-133A platform.
Clinical data for the samples in \citet{Gatza2010a} study was not available, as these samples were a combination of many different datasets.
(Find out where the 18 pathway signatures were downloaded from)

%TODO: Find Cris's paper

The raw microarray gene expression data files from the (Cris' paper) study were downloaded from the \gls{geo} database (\gls{geo} accession ID: GSE36771).
Clinical data for the samples (age, ethnicity, tumour grade, breast cancer subtype, \gls{er}/\gls{pr} statuses, \gls{ln} status, \gls{bmi} and treatments used) in this study was taken from (Cris' paper).

All of the microarray data associate the genes with the corresponding gene probe IDs, and therefore these IDs had to be converted into their corresponding gene symbols.
The gene probe IDs in the raw data were converted into their corresponding gene symbols using the \textit{hgu133a.db} package in \gls{r} \citep{hgu133}.
Since multiple gene probes matches back to a single gene of interest in a microarray chip, there were conflicting expression data for some of the genes after the conversion of the gene probe IDs into gene symbols.
For the gene symbols that had multiple data entries, a single data was chosen to represent the gene symbol by using the \texttt{collapseRows} function in the \textit{WGCNA} package in \gls{r} \citep{Langfelder2008}.
Likewise, any obesity associated or pathway associated gene probes were converted into gene symbols.
\\

% TODO: Overall summary table of the samples from each data set at the end of this section (?).

\noindent
The clinical data for all available cancer types (33 types in total) were downloaded from \gls{tcga} database (last accessed 1 April 2015) and were checked for both the height and weight data for each sample.
Any cancer type with no height and/or weight data of the samples were excluded from the project, as no \gls{bmi} information can be obtained without these data.
Out of these 33 cancer types with clinical data, 14 cancer types had both height and weight data.
However, only 8 cancer types out of these 14 types had \gls{rnaseq} data available from the \gls{icgc} database (last accessed 7 September 2015), so only those 8 cancer types were downloaded and used in this project.
The selected cancer types were: \gls{blca}, \gls{cesc}, \gls{coad}, \gls{kirp}, \gls{lihc}, \gls{read}, \gls{skcm}, \gls{ucec}.

The raw \gls{rnaseq} data from \gls{icgc} database were formatted in a way that the count data of all the genes were listed for one sample, then the count data for all the genes for the next sample, and so on.
This data format was highly inconvenient for later analyses so the data were reformatted into gene by sample matrix using the \textit{dplyr} package in \gls{r}.

Another problem with the data was the sample ID in the \gls{rnaseq} data.
Though similar to the \gls{tcga} sample IDs, the \gls{icgc} IDs in the \gls{rnaseq} data had extra identification code in the sample names.
To associate each sample in the raw \gls{rnaseq} data from the \gls{icgc} database with the correct samples in the clinical data from the \gls{tcga} database, the \gls{icgc} IDs were stripped so they matched the \gls{tcga} IDs.
After the IDs were stripped, all of the samples were checked to see if there were any duplicates in either the \gls{icgc} \gls{rnaseq} data or \gls{tcga} clinical data.
Where there was a duplicate in the sample ID, a single sample was chosen to represent that particular ID by using the \texttt{collapseRows} function in the \textit{WGCNA} package in \gls{r} \citep{Langfelder2008}.

Since some of the samples did not have either height or weight data, or both in the clinical data, these samples were removed from the analyses.
In each of the eight cancer types, the \gls{tcga} sample ID in the clinical data were cross-checked with the sample ID in the \gls{rnaseq} data, and vice versa.
Any sample that did not have either clinical information or \gls{rnaseq} data were removed from the analyses.
This ensured that both clinical and \gls{rnaseq} data were available for all of the samples that were included in the study.
See \cref{tab:samplesize} for a summary of the number of samples included in the analyses for each cancer type.

%TODO: May need more details on the sample (bmi status, etc.)
\begin{table}[h]
	\caption{Summary of the total number of samples included in the analyses for each cancer type}
	\label{tab:samplesize}
	\begin{center}
		\begin{tabular}{lc}
			Cancer Type   & Number of samples \\
			\hline
			\rule{0pt}{2.25ex}BLCA & 261   \\
			CESC                   & 224   \\
			COAD                   & 226   \\
			KIRP                   & 124   \\
			LIHC                   & 264   \\
			READ                   & 73    \\
			SKCM                   & 218   \\
			UCEC                   & 482   \\
			\hline
			\hline
		\end{tabular}
	\end{center}
\end{table}


\section{Data processing}
\label{sec:datproc}

\subsection{Data normalisation}
\label{sub:data_normalisation}

All of the data were normalised to remove any experimental bias, errors and noise, so only the true biological signals are considered in the analyses.
Any experimental procedure is prone to errors due to differences in experimental conditions, machinery used to measure signals and technical procedures in different laboratories, just to name a few.
In order to remove these experimental noises and focus on the true biological signals in the raw data, the data must be normalised one way or another.

\subsubsection{Microarray data}
\label{ssub:microarray_data}

In microarray experiments, there are two types of probes present on the microarray chips: \gls{pm} and \gls{mm} probes \citep{Irizarry2003}.
As the name suggests, \gls{pm} probes represent the probes that should perfectly match the gene of interest, whereas \gls{mm} probes have there 13th base pair intentionally altered to measure non-specific binding of the gene \citep{Irizarry2003}.
Various normalisation methods make use of \gls{pm} and \gls{mm}, or ``probe pairs'', to identify true signals from the noise.

\Gls{mas} and \gls{rma} normalisation methods are both part of the \textit{affy} package in \gls{r} \citep{Gautier2004}.
\Gls{mas} uses a difference-based method where it subtracts a value derived from \gls{mm} from \gls{pm}, but this approach may introduce additional noise and/or errors in some cases \citep{Irizarry2003}.
\Gls{rma} method is based on the observation that \gls{pm} is a mixture of background and true signal, and uses mathematical models to estimate the expression while correcting for background signals from \gls{pm} only \citep{Irizarry2003}.
In fact, the \gls{rma} method showed better identification of the true signals than the other methods, including \Gls{mas} method \citep{Irizarry2003}.

The raw microarray data were normalised using both \gls{rma} and \gls{mas} methods, each done separately on a copy of the data.
The reason why \gls{mas} method was used as well as the \gls{rma} method was becuase the \gls{mas} normalisation method was used in \citet{Gatza2010a} study.
Though the normalisation methods were not specified in the \citet{Creighton2012} and \citet{Fuentes-Mattei2014} studies, to allow for data comparison between different datasets and accurate result validation of some of the studies, both \gls{rma} and \gls{mas} methods were used to normalise the microarray data.

\subsubsection{RNA-seq data}
\label{ssub:rna_seq_data}

\gls{rnaseq} data are fundamentally different to the microarray data, as \gls{rnaseq} data is a count data of all the sequences, while microarray data is a continuous data of the intensity of the matching probes.
This means that \gls{rnaseq} data must be processed in a different manner as the microarray data.
However, by analysing \gls{rnaseq} data as a count data, it limits range of statistical tools and types of analyses that can be done on the data, as many tools were designed for normally distributed data \citep{Law2014}.

To overcome this limitation, \citet{Law2014} developed a method called ``variance modelling at the observational level'', or voom, that allows any \gls{rnaseq} data to be used in any existing statistical analysis pipeline that is precision weight aware, including pipelines used for microarray data analyses.
In brief, voom first constructs a standard deviation trend from the logged \gls{cpm} value of the genes (experimental design, treatment conditions and other factors are taken into account).
This trend is then used to interpolate the standard deviation of the observation based on its predicted count size, and the inverse square of the predicted standard deviation is used as the weight  for that observations.
The weights of the observations and the logged \gls{cpm} can then be used in other statistical pipelines that allows the input of quantitative weights \citep{Law2014}.

The raw \gls{rnaseq} data were normalised in two different ways, depending on the analysis.
For gene expression analyses, voom normalisation method from the \textit{limma} package in \gls{r} was used to normalise the data \citep{Ritchie2015}.
For the purposes of data visualisation or application of metagene transformation matrices on the \gls{rnaseq} data, the raw data had 1 added (to prevent logging of 0) and then logged to the base of 10.

\subsection{Data standardisation}
\label{sub:data_standardisation}

For metagene creation and data visualisation that used heatmaps, the data were standardised so that each gene in the data had a \gls{m} of 0 and a \gls{sd} of 1.
Since the expression levels of the genes could vary significantly (some may have very low expression, whereas another may have very high expression), the direct comparison of the raw expression values between different genes was not feasible.
Standardisation of each gene allowed the expression levels to be within acceptable range or scale, thus allowed for better visualisation with heatmaps and comparison between different genes was made possible.

\subsection{Residual data creation}
\label{sub:residual_data_creation}

In order to analyse the data without the effect of certain clinical variables on the data, the residual data of the data was used in some analyses.
For example, a gene from the raw data might have a strong association with \gls{bmi}, but it is possible that other clinical variables such as \gls{er} status and/or tumour grades are also associated with the gene.
To focus only on the association with certain variables, the effect of those variables that may affect the analyses must be removed from the data.

To do this, a linear model was created from the data with all of the clinical variables that had to be controlled for; in other words, all of the clincial variables apart from the variables of interest were included in the linear model.
Once a linear model was fitted to the data, the remaining data, or the ``residual data'', represented the data that had been corrected for all of the unwanted variables.
This data was used in some analyses that required focus on certain variables without the other variables affecting the result.

\subsection{Batch correction}
\label{sub:batch_correction}

It is most likely that experiments are done at different time period, location and laboratory environment.
These differences between experiments introduces systematic non-biological differences or ``batch effects'' into the data, making it difficult to directly compare the data from different batches \citep{Johnson2007}.
The problem with the batch effect is that the normalisation methods do not control and adjust for the effect \citep{Johnson2007}.
Fortunately, \citet{Johnson2007} developed a method to correct for batch effects in the data, using an empirical Bayesian framework.

Since the data used in \citet{Gatza2010a} study was a combination of multiple microarray data from various studies, the batch effect had to be corrected before any analysis was carried out.
Each microarray dataset was normalised separately then combined together into a single dataset, and the batch effect was corrected with the \texttt{ComBat} function (an implementation of the batch correcting method by \citet{Johnson2007}) from the \textit{sva} package in \gls{r} \citep{Leek2012}.

\section{Gene expression analysis}
\label{sec:gene_expression_analysis}

For gene expression analysis, or differential expression analysis, the \textit{limma} package in \gls{r} was used \citep{Ritchie2015}.
Since there were thousands of genes to be hypothesis tested, adjustment for multiple hypothesis testing had to be considered in order to identify the truly \glspl{deg}.

\subsection{Limma}
\label{sub:limma}

\textit{limma} is a package that contains variety of tools to analyse microarray data using linear models-based methods, developed by \citet{Ritchie2015}.
For gene expression analysis, \textit{limma} package fits a linear model in a genewise manner and produce test statistics that allow the assessment of whether the gene is differentially expressed or not \citep{Ritchie2015}.

Before the data was analysed using the \textit{limma} analysis pipeline, it was normalised as described in \cref{sub:data_normalisation}: \nameref{sub:data_normalisation}.
In addition to this, a design matrix that describes the experimental design was created from the clinical data.
To create the design matrix, the samples were divided into two groups, the obese group and the non-obese group, and the constructed group information was used in the \texttt{model.matrix} function (in \textit{limma} package) to form the design matrix.

\texttt{lmFit} function (\textit{limma}) was used to fit a linear model to the normalised data, using the experiental design information from the design matrix.
The output of this function was used in the \texttt{eBayes} function (\textit{limma}) to identify the \glspl{deg} from the data.
In \texttt{eBayes} function, statistical parameters are estimated from the data and these parameters are used in the empirical Bayesian approach to calculate the summary statistics used for the ranking and identification of \glspl{deg} \citep{Smyth2004}.

The summary statistics from the \texttt{eBayes} function can be displayed with the \texttt{topTable} function (\textit{limma}) and includes: the estimate of the fold change of the gene expression in log$_2$, average gene expression level in log$_2$, moderated \textit{t}-statistic, raw p-value of the gene, multiple hypothesis testing adjusted p-value, and \textit{B}-statistic.
The first value shows the estimate of the log$_2$ fold change of the gene expression compared with the reference group, so this represents the log$_2$ fold change of the gene expression in the obese samples relative to the non-obese group.
Second value presents the log$_2$ value of the average expression of the gene across all of the arrays/samples.
Moderated \textit{t}-statistic is the same as normal \textit{t}-statistic, but its standard error has been adjusted by a simple Bayesian method \citep{Smyth2005}.
Raw p-value and adjusted p-value represent the p-value of the gene before and after it has been corrected for multiple hypothesis testing, respectively.
Lastly, the value of \textit{B}-statistic represents the log-odds that the gene is differentially expressed, where \textit{B}-statistic of 0 shows that there is a 50\% chance the gene is differentially expressed \citep{Smyth2005}.

From these summary statistics, the most likely \glspl{deg} were chosen from the list of significant genes by setting the threshold of the p-value to be either less than 1\% or 5\%.
In the case where there were more than 1000 \glspl{deg}, the top 799 genes were picked from the list, as this was the number of genes found in the \citet{Creighton2012} study; otherwise, as many significant genes identified were taken from the list.

\subsection{Multiple hypothesis testing correction}
\label{sub:multiple_hypothesis_testing_correction}

In an experiment where there are multiple hypotheses being tested, the rate or proportion of \gls{type1} appearing from the experiment must be controlled.
The probability of \gls{type1} occuring for a single hypopthesis is usually controlled at some significance threshold $\alpha$, which is usually set at 0.05 \citep{Shaffer1995}.
In a typical microarray experiment, there are over 20,000 gene probes to be tested for differential expression, and with a significance threshold of $\alpha = 0.05$, this would yield approximately 1,000 \glspl{type1}.
In other words, 1,000 gene probes would be identified as differentially expressed, when in fact they are not.

There are two broad classes of methods to correct for this problem: \gls{fwer} control and \gls{fdr} control.
With \gls{fwer} control, the method primarily aims to set the $\alpha$-value for each hypothesis testing ($\alpha_i$) such that the sum of all the $\alpha_i$ is equal to $\alpha$ \citep{Hochberg1987,Shaffer1995}.
Usually, $\alpha_i$ is set to $\frac{\alpha}{n}$, where $n$ is the number of hypothesis tests carried out in the experiment \citep{Shaffer1995}.
This highly conservative approach significantly improves the certainty of the result from the experiment, but at the same time it significantly increases the likelihood of missing the truly \glspl{deg}, or \glspl{type2}.

In contrast to the conservative \gls{fwer} control methods, the \gls{fdr} method developed by \citet{Benjamini1995a} control the \glspl{type1} while maintaining statistical \gls{power}.
The \gls{fdr} method controls the ``expected proportion of errors among the rejected hypotheses'' by adjusting the $\alpha$-level (denoted as $q^*$ in \gls{fdr}) depending on the rank of the p-value \citep{Benjamini1995a}.
With \gls{fdr}, the p-values are ordered and ranked from the lowest to the highest p-value, and for each hypothesis the p-value is compared with the adjusted threshold value:
\begin{equation}
	\label{eq:fdr}
	P_{(i)} \leq \frac{i}{m}q^*
\end{equation}
where $P_{(i)}$ is the p-value of the ordered and ranked $i$th hypothesis, $m$ is the total number of hypotheses, and $q^*$ is the threshold value \citep{Benjamini1995a}.
From \cref{eq:fdr}, the adjusted p-value from \gls{fdr} control is the product of the p-value with the number of hypotheses, divided by its rank.

Since \gls{fdr} method controls \gls{type1} and have greater statistical \gls{power} than the \gls{fwer} methods, \gls{fdr} method was used to identify the \glspl{deg} for gene expression analysis.

\section{Pathway enrichment analysis}
\label{sec:pathway_enrichment_analysis}

Identification of \glspl{deg} provides great deal of information about the effect of treatment and provides a list of genes that may have a role in that particular experimental setting.
With that said, it is often difficult for the investigators to provide a plausible biological explanation with just a long list of \glspl{deg}, as the list lacks the link between the genes with the biological cause \citep{Khatri2012}.
Therefore, given a list of \glspl{deg}, it is crucial for any researcher to undertake pathway enrichment analysis in order to provide useful insights into the underlying biological mechanisms the \glspl{deg} may be involved in.

\subsection{Competitive and self-contained tests}
\label{sub:competitive_and_self_contained_tests}

In general, there are two major classes of methods for pathway enrichment analysis: self-contained or competitive tests \citep{Goeman2007}.
The main difference between the two classes is the definition of the null hypothesis.
Letting $G$ be the gene set of interest and $G^c$ its compliment, then in self-contained tests, the null hypothesis is formulated as follows:

\begin{quote}
	\textit{$H_0^{\textrm{self}}$: No genes in G are differentially expressed. }
\end{quote}

\noindent
whereas in competitive tests, the null hypothesis is defined as:

\begin{quote}
	\textit{$H_0^{\textrm{comp}}$: The genes in G are at most as often differentially expressed as the genes in $G^c$.
	}
\end{quote}

These hypotheses show that self-contained tests care only about the genes defined in the gene set, but competitive tests examine the genes in the defined set as well as the genes not present in the set \citep{Wu2012}.
Due to the fact that self-contained tests do not take into account for the genes not defined in the gene set, self-contained tests are more restrictive compared to competitive tests.
The restrictive property of self-contained tests give it greater power, as they are able to reject the null hypothesis at a higher accuracy for more gene sets than the competitive tests \citep{Goeman2007}.
As \citet{Goeman2007} stated in their paper, ``the competitive types of test can be said to voluntarily relinquish some power in order to make a stronger statement''.

Taken together, self-contained tests are more appropriate in assessing whether a biological process is significantly involved in an expreiment, and in contrast, competitive tests are better for selecting the most relevant biological processes from those that are not \citep{Wu2012}.

\subsection{\Gls{ora} and \gls{fcs}}
\label{sub:ora_and_fcs}

Though the approaches differ between the two, both \acrfull{ora} and \acrfull{fcs} are used to identify whether certain pathways are enriched, given a list of \glspl{deg}.
The common approach taken by \gls{ora} is to count the \glspl{deg} that are part of a biological pathway and perform a statistical test to decide whether the pathway is over- or under-represented in the list of genes \citep{Khatri2012}.
Statistical test used in \gls{ora} includes $\chi^2$-test, hypergeometric test, and binomial test \citep{Khatri2012}.
Depending on the methods, \gls{ora} uses either self-contained or competitive tests.

There are several limitations to the \gls{ora} approach.
Firstly, the statistical tests are independent of the measured changes and therefore ignores the values associated with the genes, such as intensities and significance of the change \citep{Khatri2012}.
Secondly, only the most significant genes are selected for the input list of genes, which means that the almost significant genes are discarded from the analysis and results in a loss of information \citep{Khatri2012}.
Lastly, by treating each gene individually, the analysis ignores the biological interaction and complexity of the genes with other genes, as well as between pathways \citep{Khatri2012}.

In general, \gls{fcs} methods measure gene-level statistics from the given data, then aggregate these statistics into pathway-level statistics, which is then used to calculate the statistical significance of the pathway \citep{Khatri2012}.
Either self-contained or competitive tests can be used to calculate the statistical significance of the pathways.

\Gls{fcs} approach addresses some of the limitations presented by the \gls{ora} methods.
By calculating the statistical significance per pathway, \gls{fcs} approach takes into account of all of the genes involved in the pathways, not just the genes that are differentially expressed.
Furthermore, \gls{fcs} approach also detect small but consistant and coordinated changes in the expression of the genes, unlike in \gls{ora} where molecular measurements are completely ignored \citep{Khatri2012}.

Though \gls{fcs} approach improves on \gls{ora} approach, there are still some limitations.
Since the analyses that use \gls{fcs} compare the pathways independently from one another, it ignores the interactive nature of biological pathways.
Another limitation is that \gls{fcs} methods use the molecular measurements to rank the genes, but they do not consider these changes into further analysis \citep{Khatri2012}.
For example, if one gene was expressed with a 2-fold change, whereas another gene was expressed with a 20-fold change, the \gls{fcs} method will rank these genes accordingly and disregard the fact that the second gene should be weighted more than the first gene \citep{Khatri2012}.

Although both approaches have their own limitations, it is clear that \gls{fcs} approach have more advantages over \gls{ora} methods.

\subsection{\Gls{camera}}
\label{sub:camera}

\Acrfull{camera} is a competitive \gls{fcs}-based method developed by \citet{Wu2012}, implemented as the \texttt{camera} function in the \textit{limma} package \citep{Ritchie2015}.
Briefly, \gls{camera} fits a linear model in a genewise manner and calculates the genewise test statistics using the log\gls{fc} between the conditions.
These genewise test statistics are used in the \gls{wmw} rank sum test to question whether a pathway is significantly enriched in the data \citep{Wu2012}.

One problem with the other \gls{fcs}-based methods is that these methods do not consider the inter-gene correlation of the gene set being tested \citep{Wu2012}.
Since other methods assume that all the genes are equivalent under the null hypothesis, inter-gene correlation of the genes in a gene set will violate this assumption \citep{Wu2012}.
As a consequence, the \gls{type1} rate is increased in these methods.
\Gls{camera} accounts for this correlation by estimating the \gls{vif} from the genewise correlation and the number of genes in the gene set \citep{Wu2012}.

\gls{camera} was used in this project to identify the pathways that are enriched in samples from obese patients compared with the samples from non-obese patients, as it provides a competitive \gls{fcs}-based method.

\subsection{\gls{go} database}
\label{sub:go_database}













For a given list of differentially expressed genes identified in a data set, the \texttt{camera} function from the \textit{limma} package was used to identify the pathways that were enriched in those genes.
The \gls{go} database was used to identify the enriched pathways.

\section{Metagene analysis}
\label{sec:metagene_analysis}

\gls{svd} was applied to the training data set to create the metagene and the transformation matrix in the data set.
The transformation matrix from the training data was used to transform the other data sets and to create the metagene of the genetic signature in those data sets.

Each of these metagenes were plotted in a heatmap to see the association between the metagene and the overall gene expression of the genes that were used to create the metagene.
The metagene was also checked for the association between the sample \gls{bmi} status and \gls{bmi} value, which was plotted in a box plot and a scatter plot, respectively.

% \section{\Gls{svd}}
% \label{sec:singular_value_decomposition}

\gls{svd} was used on a given data set to create the summary metagene scores of a given genetic signature.
% TODO: add more details on the components (s, v, d) created when svd was applied to the data.
Any matrix $X$ can be represented in the form:
\begin{equation}
	\label{eq:svd}
	X = UDV'
\end{equation}

\noindent
where $U$ is the *****, $D$ is the ********, and $V'$ is the **** (eigenvectors and eigenvalues??).

% \section{Transformation matrix}
% \label{sec:transformation_matrix}

Transformation matrix of a genetic signature was created from the \gls{svd} components.
Rearranging \cref{eq:svd}, you get:

\begin{equation}
	\label{eq:transmat}
	V' = U'D^{-1}X
\end{equation}

\noindent
where $U'D^{-1}$ represents the transformation matrix.
The transformation matrix $U'D^{-1}$ was created from the \gls{svd} components, and was used to transform other data set and obtain the metagene in the data set.

\section{Gatza pathway metagene direction}
\label{sec:pathway_metagene_direction}

All 18 pathway signatures from \citet{Gatza2010a} study were used to create both the metagenes and transformation matrices in the Gatza data set.
Each of the 18 pathway metagenes were compared with the gene expression of the gene that represents the pathway to check whether the metagenes were in the correct direction.
% TODO: check if it was the AKT1 or AKT2 gene for AKT pathway
For example, the AKT pathway metagene was compared with the AKT1 gene expression.

The pathway metagene value was first compared visually with its pathway gene in a heatmap.
If the direction of the pathway metagene was correct (for example, high pathway metagene value corresponded with high expression of the gene representing that pathway), the metagene values were not changed, but if it was incorrect, the metagene values were flipped for that pathway.

Spearman correlation of the pathway metagene and the expression of the pathway gene was also used to distinguish whether the pathway metagene was in the right direction.
The direction of the metagene was considered correct when the correlation was positive and incorrect when the correlation was negative.

The clustering of the pathway metagenes were taken into account to decide on the final direction of the metagene.
The direction of the pathway metagenes were modified until the resulting pathway metagenes were similarly clustered together as in the results from \citet{Gatza2010a} paper.

\section{Obesity metagene prediction}
\label{sec:obesity_metagene_prediction}

% TODO: possibly extra analyses here (gradual pathway inclusion in the model, using all the pathway...?)
Sample BMI, BMI status and/or pathway metagenes were used to construct a linear model in Cris Print's data.
The constructed linear model was then used to predict the obesity metagene values in \citet{Creighton2012} data set.
The predicted obesity metagene values were compared with the true obesity metagene values, and both Spearman and Pearson correlation were calculated.

\section{Plot creation}
\label{sec:plot_creation}

\subsection{Heatmaps}
\label{sub:heatmaps}

Heatmaps were created using either the \texttt{heatmap.2} function from \textit{gplots} package, or the \texttt{heatmap.2x} function written by Tom Kelly.
All of the heatmaps that required no or one column bar used the \texttt{heatmap.2} function, and all of the heatmaps that required more than one column bars used the \texttt{heatmap.2x} function.




% TODO: re-write everything so that it's in past tense
