\chapter{Methods (draft)}
\label{ch:methods}

\section{\gls{bmi}}
\label{sec:bmi}

\subsection{\gls{bmi} Calculation}
\label{subsec:bmicalc}

\cref{eq:bmicalc}  was used to calculate the \gls{bmi} from the height and weight data of the samples.

\begin{equation}
	\label{eq:bmicalc}
	\gls{bmi} = \frac{Weight (kg)}{Height^2(m^2)}\\
\end{equation}

\subsection{\gls{bmi} Classification}
\label{subsec:bmiclassification}

Samples were classified based on the \gls{who} definition, as shown in \cref{tab:whobmiclass}.
\begin{table}[hb]
	\caption{\gls{who} defined \gls{bmi} classification}
	\label{tab:whobmiclass}
	\begin{center}
		\begin{tabular}{lc}
			\bfseries {Classification} & \bfseries {\gls{bmi} Value}\\
			\hline
			\rule{0pt}{2.25ex}Underweight & \textless{} 20.0\\
			Normal weight/lean & 20.0$\sim$24.9\\
			Overweight & 25.0$\sim$29.9\\
			Obese & $\geq{}$ 30.0\\
		\end{tabular}
	\end{center}
\end{table}

\section{Publicly available cancer data}
\label{sec:data}

% TODO: add Cris Print's paper
The raw microarray data from \citet{Creighton2012}, \citet{Fuentes-Mattei2014} and \citet{Gatza2010a}  studies were downloaded from the \gls{geo} website.
\gls{rnaseq} and clinical data of multiple different cancer types were downloaded from free and publicly available sources such as \gls{icgc} and \gls{tcga}, respectively.

\subsection{Raw data from \citet{Creighton2012} study}
\label{subsec:crrawdat}

The raw microarray gene expression data files from the \citet{Creighton2012} study were downloaded from the \gls{geo} database (\gls{geo} accession ID: GSE24185).
Clinical data of the samples were obtained from the supplementary table 1 from \citet{Creighton2012} paper.
799 obesity associated gene probes identified in the \citet{Creighton2012} study were obtained from the supplementary data file 1 from \citet{Creighton2012} paper.
The raw expression data, clinical data, and the list of gene probes were imported into \gls{r}.

\subsection{Raw data from \citet{Fuentes-Mattei2014} study}
\label{subsec:fmrawdat}

The raw microarray gene expression data files from the  \citet{Fuentes-Mattei2014} study were downloaded from the \gls{geo} database (\gls{geo} accession ID: GSE\-20194).
Clinical data for the samples in this study was also downloaded from the \gls{geo} database (same \gls{geo} accession ID).
130 obesity associated gene probes identified by \citet{Fuentes-Mattei2014} were taken from the supplementary table 3.
The raw expression data and the list of gene probes were imported into \gls{r}.

\subsection{Raw data from \citet{Gatza2010a} study}
\label{sub:gatzarawdat}

The raw microarray gene expression data files from the \citet{Gatza2010a} study were downloaded from the \gls{geo} database (\gls{geo} accession ID: GSE1456, GSE\-1561, GSE2034, GSE3494, GSE4922, and GSE6596).
Only the Affymetrix HGU\-133A microarray samples were included in the project.
Clinical data for the samples in this study was not available.
The raw expression data and the list of gene probes were imported into \gls{r}.

\subsection{Raw data from Print et al study}
\label{subsec:printrawdat}

%TODO: Find Cris's paper

The raw microarray gene expression data files from the  Print et al. study were downloaded from the \gls{geo} database (\gls{geo} accession ID: GSE36771).
Clinical data for the samples in this study was ******.
The raw expression data and the list of gene probes were imported into \gls{r}.

% TODO: Overall summary table of the samples from each data set at the end of this section.

\subsection{\gls{tcga}}
\label{subsec:tcga}

The clinical data for all available cancer types (33 types in total) were downloaded from \gls{tcga} database and were checked for the height and weight data for each sample.
Any cancer type with no weight and/or height data of the samples were excluded from the project, as no \gls{bmi} information can be obtained without these data.

This gave a total of eight cancer types: \gls{blca}, \gls{cesc}, \gls{coad}, \gls{kirp}, \gls{lihc}, \gls{read}, \gls{skcm}, \gls{ucec}.
All eight cancer types were used for further analyses.

\subsection{\gls{icgc}}
\label{subsec:icgc}

\gls{rnaseq} data for the eight cancer types from \cref{subsec:tcga} were downloaded from the \gls{icgc} database.
The \gls{rnaseq} data for each of the eight cancer types were imported into \gls{r}/RStudio.

\section{Data processing}
\label{sec:datproc}

\subsection{Data normalisation}
\label{sub:data_normalisation}

\subsubsection{Microarray data}
\label{ssub:microarray_data}

The raw microarray data were normalised using either the \gls{rma} or \gls{mas} method from the \textit{affy} package in \gls{r}.
Either \gls{rma} or \gls{mas} normalisation was used to account for the inter-array differences that may have arisen during the data collection process.

\subsubsection{RNA-seq data}
\label{ssub:rna_seq_data}

The raw \gls{rnaseq} data were normalised in two different ways, depending on the analysis.
For gene expression analyses, \texttt{voom} function from the \textit{limma} package was used to normalise the data.
For the purposes of data visualisation or application of metagene transformation matrices, the data had 1 added and then logged to the base of 10.

\subsection{Data standardisation}
\label{sub:data_standardisation}

The data was standardised so that each gene in the data had a mean ($\mu$) of 0 and a standard deviation ($\sigma$) of 1.
This ensured that all of the genes were on the same scale as one another, which allowed direct comparison across all of the genes.

\subsection{Residual data creation}
\label{sub:residual_data_creation}

A linear model was constructed from the data with all of the clinical variables controlled for.
The residual data was taken from this linear model to be used as the residual data for that data set.

\subsection{Gene probe ID conversion}
\label{sub:gene_probe_id_conversion}

Where appropriate, the gene probe IDs in the raw data were converted into their corresponding gene symbols using the \textit{hgu133a.db} package in \gls{r}.
Where there was an overlap in the gene symbols, a single data was chosen to represent that particular gene symbol by using the \texttt{collapseRows} function in the \textit{WGCNA} package in \gls{r}.
Likewise, the obesity associated gene probes were also converted into gene symbols where required.


\subsection{Batch correction}
\label{sub:batch_correction}

When there were more than one microarray data to be used as a single data set, these microarray data were normalised separately then combined together with the \texttt{ComBat} function in the \textit{sva} package in \gls{r}.
This corrected for the effect of the individual microarray data on the whole data when combined together.

\subsection{Sample \gls{bmi} calculation}
\label{sub:sample_bmi_calculation}

Where missing, the \gls{bmi} of the samples were calculated from the clinical data using \cref{eq:bmicalc}.

\subsection{\gls{icgc} data format conversion}
\label{sub:icgc_data_format_conversion}

The raw \gls{rnaseq} data for each cancer types were formatted using the \textit{dplyr} package in \gls{r}, so that the rows corresponded to the genes and the columns to the samples.

\subsection{Sample ID conversion}
\label{sub:sample_id_conversion}

\gls{icgc} sample ID in the \gls{rnaseq} data were stripped so that the sample ID matched with the \gls{tcga} ID in the clinical data.
After the IDs were stripped, samples were checked for any duplicates.
Where there was a duplicate in the sample ID in the \gls{rnaseq} data, a single sample was chosen to represent that particular ID by using the \texttt{collapseRows} function in the \textit{WGCNA} package in \gls{r}.

\subsection{\gls{icgc} sample exclusion}
\label{sub:icgc_sample_exclusion}

Some of the samples did not have either height or weight data, or both.
These samples were removed from the analyses.
In each of the eight cancer types, the \gls{tcga} sample ID in the clinical data were cross-checked with the sample ID in the \gls{rnaseq} data, and vice versa.
Any sample that did not have either clinical or \gls{rnaseq} data were removed.
This ensured that both clinical and \gls{rnaseq} data were available for all of the samples that were included in the analyses.
See \cref{tab:samplesize} for a summary of the number of samples included in the analyses for each cancer type.

%TODO: May need more details on the sample (bmi status, etc.)
\begin{table}[h]
	\caption{Summary of the total number of samples included in the analyses for each cancer type}
	\label{tab:samplesize}
	\begin{center}
		\begin{tabular}{cc}
			\textbf{Cancer Type}   & \textbf{Total number of samples} \\
			\hline
			\rule{0pt}{2.25ex}BLCA & 261   \\
			CESC                   & 224   \\
			COAD                   & 226   \\
			KIRP                   & 124   \\
			LIHC                   & 264   \\
			READ                   & 73    \\
			SKCM                   & 218   \\
			UCEC                   & 482   \\
		\end{tabular}
	\end{center}
\end{table}

\section{Gene expression analysis}
\label{sec:gene_expression_analysis}

The samples were grouped based on the sample BMI status (obese or non-obese group), and a linear model was fitted to the data based on the sample grouping.
Empirical Bayes statistics was used to identify the differentially expressed genes between the two groups, and the top scoring genes were listed.
The error rate of the genes were \gls{fdr} controlled for multiple hypothesis testing.

\section{Pathway enrichment analysis}
\label{sec:pathway_enrichment_analysis}

For a given list of differentially expressed genes identified in a data set, the \texttt{camera} function from the \textit{limma} package was used to identify the pathways that were enriched in those genes.
The \gls{go} database was used to identify the enriched pathways.

\section{\gls{svd}}
\label{sec:singular_value_decomposition}

\gls{svd} was used on a given data set to create the summary metagene scores of a given genetic signature.
% TODO: add more details on the components (s, v, d) created when svd was applied to the data.
Any matrix $X$ can be represented in the form:
\begin{equation}
	\label{eq:svd}
	X = UDV'
\end{equation}

\noindent
where $U$ is the *****, $D$ is the ********, and $V'$ is the **** (eigenvectors and eigenvalues??).

\section{Transformation matrix}
\label{sec:transformation_matrix}

Transformation matrix of a genetic signature was created from the \gls{svd} components.
Rearranging \cref{eq:svd}, you get:

\begin{equation}
	\label{eq:transmat}
	V' = U'D^{-1}X
\end{equation}

\noindent
where $U'D^{-1}$ represents the transformation matrix.
The transformation matrix $U'D^{-1}$ was created from the \gls{svd} components, and was used to transform other data set and obtain the metagene in the data set.

\section{Metagene analysis}
\label{sec:metagene_analysis}

\gls{svd} was applied to the training data set to create the metagene and the transformation matrix in the data set.
The transformation matrix from the training data was used to transform the other data sets and to create the metagene of the genetic signature in those data sets.

Each of these metagenes were plotted in a heatmap to see the association between the metagene and the overall gene expression of the genes that were used to create the metagene.
The metagene was also checked for the association between the sample \gls{bmi} status and \gls{bmi} value, which was plotted in a box plot and a scatter plot, respectively.

\section{Gatza pathway metagene direction}
\label{sec:pathway_metagene_direction}

All 18 pathway signatures from \citet{Gatza2010a} study were used to create both the metagenes and transformation matrices in the Gatza data set.
Each of the 18 pathway metagenes were compared with the gene expression of the gene that represents the pathway to check whether the metagenes were in the correct direction.
% TODO: check if it was the AKT1 or AKT2 gene for AKT pathway
For example, the AKT pathway metagene was compared with the AKT1 gene expression.

The pathway metagene value was first compared visually with its pathway gene in a heatmap.
If the direction of the pathway metagene was correct (for example, high pathway metagene value corresponded with high expression of the gene representing that pathway), the metagene values were not changed, but if it was incorrect, the metagene values were flipped for that pathway.

Spearman correlation of the pathway metagene and the expression of the pathway gene was also used to distinguish whether the pathway metagene was in the right direction.
The direction of the metagene was considered correct when the correlation was positive and incorrect when the correlation was negative.

The clustering of the pathway metagenes were taken into account to decide on the final direction of the metagene.
The direction of the pathway metagenes were modified until the resulting pathway metagenes were similarly clustered together as in the results from \citet{Gatza2010a} paper.

\section{Obesity metagene prediction}
\label{sec:obesity_metagene_prediction}

% TODO: possibly extra analyses here (gradual pathway inclusion in the model, using all the pathway...?)
Sample BMI, BMI status and/or pathway metagenes were used to construct a linear model in Cris Print's data.
The constructed linear model was then used to predict the obesity metagene values in \citet{Creighton2012} data set.
The predicted obesity metagene values were compared with the true obesity metagene values, and both Spearman and Pearson correlation were calculated.

\section{Plot creation}
\label{sec:plot_creation}

\subsection{Heatmaps}
\label{sub:heatmaps}

Heatmaps were created using either the \texttt{heatmap.2} function from \textit{gplots} package, or the \texttt{heatmap.2x} function written by Tom Kelly.
All of the heatmaps that required no or one column bar used the \texttt{heatmap.2} function, and all of the heatmaps that required more than one column bars used the \texttt{heatmap.2x} function.




% TODO: re-write everything so that it's in past tense
