\chapter{Methods (draft)}
\label{ch:methods}

\section{\gls{bmi}}
\label{sec:bmi}

\subsection{\gls{bmi} Calculation}
\label{subsec:bmicalc}

\gls{bmi} was calculated from the height and weight data of the samples using the following equation:

\begin{equation}
	\label{eq:bmicalc}
	\gls{bmi} = \frac{Weight (kg)}{Height^2(m^2)}\\
\end{equation}

\subsection{\gls{bmi} Classification}
\label{subsec:bmiclassification}

Samples were classified based on the \gls{who} definition, as shown in Table~\cref{tab:whobmiclass}.
\begin{table}[hb]
	\caption{\gls{who} defined \gls{bmi} classification}
	\label{tab:whobmiclass}
	\begin{center}
		\begin{tabular}{lc}
			\bfseries {Classification} & \bfseries {\gls{bmi} Value}\\
			\hline
			\rule{0pt}{2.25ex}Underweight & \textless{} 20.0\\
			Normal weight/lean & 20.0$\sim$24.9\\
			Overweight & 25.0$\sim$29.9\\
			Obese & \textgreater{} 30.0\\
		\end{tabular}
	\end{center}
\end{table}

\section{Publicly available cancer data}
\label{sec:data}

The original raw data from the \citet{Creighton2012} and \citet{Fuentes-Mattei2014} studies were downloaded from the \gls{geo} website.
Data related to multiple different cancer types were downloaded from free and publicly available sources such as \gls{tcga} and the \gls{icgc}.

\subsection{Raw data from the \citet{Creighton2012} study}
\label{subsec:crrawdat}

The raw microarray gene expression data files of all *** samples from the \citet{Creighton2012} study were downloaded from the \gls{geo} database (\gls{geo} accession ID: GSE24185).
Clinical data of the samples were obtained from the supplementary table 1 from their paper.
%TODO: add data/sample composition -- ethinicity, grade, etc -- maybe in table form
799 obesity associated gene probes identified in the \citet{Creighton2012} study were obtained from the supplementary data file 1 from their paper.
The raw expression data, clinical data, and the list of gene probes were imported into \gls{r}/RStudio.

\subsection{Raw data from the \citet{Fuentes-Mattei2014} study}
\label{subsec:fmrawdat}

The raw microarray gene expression data files of all *** samples from the  \citet{Fuentes-Mattei2014} study were downloaded from the \gls{geo} database (include website name/GEO ID here).
Clinical data for the samples in this study was not available.
130 obesity associated gene probes identified by \citet{Fuentes-Mattei2014} were downloaded from (supplementary data??).
The raw expression data and the list of gene probes were imported into \gls{r}/RStudio.

\subsection{\gls{tcga}}
\label{subsec:tcga}

The clinical data for all available cancer types (33 types in total) were downloaded from \gls{tcga} database and were checked for the height and weight data for each sample.
Any cancer type with no weight and/or height data of the samples were excluded from the project, as no \gls{bmi} information can be obtained without these data.

This gave a total of eight cancer types: \gls{blca}, \gls{cesc}, \gls{coad}, \gls{kirp}, \gls{lihc}, \gls{read}, \gls{skcm}, \gls{ucec}.
All eight cancer types were used for further analyses.

\subsection{\gls{icgc}}
\label{subsec:icgc}

\gls{rnaseq} data for the eight cancer types identified in \cref{subsec:tcga} were downloaded from the \gls{icgc} database.
The \gls{rnaseq} data for each of the eight cancer types were imported into \gls{r}.





\section{Data processing}
\label{sec:datproc}

\subsection{Creighton data}
\label{subsec:crdatproc}

% \subsubsection{\textbf{Data processing}}

\subsubsection{\textbf{Raw data normalisation}}

The raw microarray data for all 103 samples from \cref{subsec:crrawdat} were normalised using the \gls{rma} method from the \textit{affy} package in \gls{r}.
\gls{rma} normalisation was used to account for the inter-array differences that may have arisen during the data collection process.

\subsubsection{\textbf{Raw data standardisation}}

The data was standardised so that each gene had a mean ($\mu$) of 0 and a standard deviation ($\sigma$) of 1.
This ensured that all of the genes were on the same scale as one another, allowing for comparison across all the genes.

\subsubsection{\textbf{Gene probe ID conversion}}

Where appropriate, the gene probe IDs in the raw data were converted into their corresponding gene symbols using the \textit{hgu133a.db} package in \gls{r}.
Where there was an overlap in the gene symbols, a single data was chosen to represent that particular gene symbol by using the \texttt{collapseRows} function in the \textit{WGCNA} package in \gls{r}.
Likewise, the obesity associated gene probes were also converted into gene symbols where required.

\subsubsection{\textbf{Sample \gls{bmi} calculation}}

\gls{bmi} of the samples were calculated from the clinical data using \cref{eq:bmicalc}.

\subsection{Fuentes-Mattei data}
\label{subsec:fmdatproc}

% \subsubsection{\textbf{Data processing}}

\subsubsection{\textbf{Raw data normalisation}}

The raw microarray data for all *** samples from \cref{subsec:fmrawdat} were normalised using the \gls{rma} method from the \textit{affy} package in \gls{r}.
\gls{rma} normalisation was used to account for the inter-array differences that may have arisen during the data collection process.

\subsubsection{\textbf{Raw data standardisation}}

The data was standardised so that each gene had a mean ($\mu$) of 0 and a standard deviation ($\sigma$) of 1.
This ensured that all of the genes were on the same scale as one another, allowing for comparison across all the genes.

\subsubsection{\textbf{Gene probe ID conversion}}

Where appropriate, the gene probe IDs in the raw data were converted into their corresponding gene symbols using the \textit{hgu133a.db} package in \gls{r}.
Where there was an overlap in the gene symbols, a single data was chosen to represent that particular gene symbol by using the \texttt{collapseRows} function in the \textit{WGCNA} package in \gls{r}.

\subsection{\gls{tcga}}
\label{subsec:datproctcga}

\subsubsection{\textbf{Raw data normalisation}}

The raw \gls{rnaseq} data for each cancer types from \cref{subsec:tcga} had 1 added, and then logged to the power of 10.

\subsubsection{\textbf{Raw data standardisation}}

The normalised data was standardised so that each gene had a mean ($\mu$) of 0 and a standard deviation ($\sigma$) of 1.
This ensured that all of the genes were on the same scale as one another, allowing for comparison across all the genes.

\subsection{\gls{icgc}}
\label{subsec:datprocicgc}

\subsubsection{\textbf{Raw data format conversion}}
%TODO: add in how I processed the ICGC data format into genes by samples.
The raw \gls{rnaseq} data for each cancer types were formatted so that the rows corresponded to the genes and the columns to the samples.
% TODO: find out what functions I used to reformat the icgc data
This was done using the \textit{dplyr} package in \gls{r}.

\subsubsection{\textbf{Sample ID conversion}}
%TODO: add details on how I stripped the ICGC ID into TCGA ID
%TODO: check the actual ID stripping process
%TODO: check the WGCNA function and check if it uses median/mean/whatever for choosing the sample
\gls{icgc} sample ID in the \gls{rnaseq} data were stripped so that the ID matched with the \gls{tcga} ID in the clinical data.
After the IDs were stripped, samples were checked for any overlaps.
Where there was an overlap in the sample ID in the \gls{rnaseq} data, a single sample was chosen to represent that particular ID by using the \texttt{collapseRows} function in the \textit{WGCNA} package in \gls{r}.

\subsubsection{\textbf{\gls{icgc} sample exclusion criteria}}
\label{subsec:icgcsampleexclusion}

% \subsubsection{\textbf{Sample BMI availability}}

Some of the samples did not have either height or weight data, or both.
These samples were removed from the analyses.
In each of the eight cancer types, the \gls{tcga} sample ID in the clinical data were cross-checked with the \gls{tcga} sample ID in the \gls{rnaseq} data, and vice versa.
Any sample that did not have either clinical or \gls{rnaseq} data were removed.
This ensured that both clinical and \gls{rnaseq} data were available for all of the samples that were included in the analyses.
See \cref{tab:samplesize} for a summary of the number of samples included in the analyses for each cancer type.

%TODO: May need more details on the sample (bmi status, etc.)
\begin{table}[h]
	\caption{Summary of the total number of samples included in the analyses for each cancer type}
	\label{tab:samplesize}
	\begin{center}
		\begin{tabular}{cc}
			\textbf{Cancer Type}   & \textbf{Total} \\
			\hline
			\rule{0pt}{2.25ex}BLCA & 261   \\
			CESC                   & 224   \\
			COAD                   & 226   \\
			KIRP                   & 124   \\
			LIHC                   & 264   \\
			READ                   & 73    \\
			SKCM                   & 218   \\
			UCEC                   & 482   \\
		\end{tabular}
	\end{center}
\end{table}

\section{Plot creation}
\label{sec:plot_creation}

\subsection{Heatmaps}
\label{sub:heatmaps}






\subsection{Box plots}
\label{sub:box_plots}





\subsection{Scatter plots}
\label{sub:scatter_plots}






\section{Gene expression analysis}
\label{sec:gene_expression_analysis}



\section{Metagene analysis}
\label{sec:metagene_analysis}





\section{Pathway enrichment analysis}
\label{sec:pathway_enrichment_analysis}


























